make_release.py

```diff
@@ -606,6 +621,7 @@ def announce_new_release(context):
     (pull, ''),
     # >>> need internet from here
     (push, ''),
+    (push_on_pypi, 'Pushing on Pypi'),
```

Comment:
I would be nice to have a "push on pypi test server" step before, and potentially split the "push_on_pypi" step into "create_pypi_packages" and "push_on_pypi", so that we can check packages before they are uploaded. This could come later though.